packages/builder/src/components/grants/About.tsx

```diff
@@ -67,15 +67,16 @@ export default function About({
         <span className="text-[20px]">My Applications</span>
       </Box>
       <Box>
-        {props.applications.map((application) => {
+        {props.applications.map((application, index) => {
           const roundID = application?.roundID;
           const cardData = {
             application,
             roundID,
             chainId: application.chainId,
           };
+
           return (
-            <Box key={roundID} m={2}>
+            <Box key={[roundID, index].join("-")} m={2}>
```

Comment:
I am not 100% sure but you might be able to do this to avoid using the array index:

```suggestion
            <Box key={[roundID, application.id].join("-")} m={2}>
```

Reply:
seems to work